### ISSUE TITLE
Questions around manifest logos

### ISSUE THREAD

**Issue by cbeer** [ProjectMirador/mirador-design#37]
Extracted from https://github.com/ProjectMirador/mirador/issues/1794 so we don't lose it

----

We definitely want to preserve aspect ratio. The question (to me, anyway) is whether we limit horizontal width in some way to prevent horizontally-oriented logos from consuming too much topbar space (and whether we omit display of logos at some viewport or window width to gain as much topbar space as possible where topbar space is limited).

Here are a couple of examples of horizontally-oriented logos from our existing fixture list:

http://beta.biblissima.fr/iiif/manifest/ark:/43093/desc57cb76cd3739a24a9277b6669d95b5f3a590e771

![logo-biblissima-350w](https://user-images.githubusercontent.com/101482/52289325-08d4b100-292b-11e9-9ea6-f912e9af36c6.jpg)

https://gallica.bnf.fr/iiif/ark:/12148/btv1b10022508f/manifest.json

![bnf](https://user-images.githubusercontent.com/101482/52289343-0f632880-292b-11e9-9c33-0fde30636c1f.png)

Not currently in our fixture list, but another example:

https://iiif.harvardartmuseums.org/manifests/object/299843

![harvard-art-museum](https://user-images.githubusercontent.com/101482/52289789-f0b16180-292b-11e9-9329-c763deb2c58d.jpg)

**Comment by sdellis** [comment]
@ggeisler and @jvine [this IIIF issue](https://github.com/IIIF/api/issues/1760) seems related to this conversation.

It would also be great to have text-based attribution associated with the logo for alt text values. @mikeapp am I correct that this is not currently possible in IIIF Prezi 3?

**Comment by christopher-johnson** [comment]
any resource can have a `requiredStatement`[[1]](https://iiif.io/api/presentation/3.0/#31-descriptive-properties) in v3.  logo is a resource.

 [1] https://iiif.io/api/presentation/3.0/#31-descriptive-properties

**Comment by sdellis** [comment]
> any resource can have a `requiredStatement`[[1]](https://iiif.io/api/presentation/3.0/#31-descriptive-properties) in v3. logo is a resource.
> 
Thanks for the tip @christopher-johnson. But since requiredStatements MUST be "displayed", it doesn't seem viable as a way to provide alt text for a logo.

**Comment by jvine** [comment]
I think we need to keep the purpose of the logo in different contexts in mind.

1. In the metadata panel, there is a place for the logo and required statement. 
2. In the various lists of manifests (catalog, collection lists), the logo is meant as a simple attribution ("provided by..") and an additional cue to help users find and select the manifests they want. 
3. In the window header, the logo is light-weight attribution, but also to helps a user with multiple windows open quickly identify the one(s) from specific sources.

**It would also be great to have text-based attribution associated with the logo for alt text values.**

Unless we make the required statement available to all users in the context of any logo presentation, I don't believe we should present it to screen readers (but not sighted users) in places where that's not the intent of the logo presentation. 

The header bar might read like this:

` Yale Center For British Art:  Peter Gaspar Scheemakers, 1691–1781, Flemish, active in Britain (from ca. 1720), Alexander Pope, ca. 1740, Marble, Yale Center for British Art, B1977.14.29, Paintings and Sculpture `

without the full text of the required statement - so the alt text is the institution name. 

In the metadata panel and manifest lists, the name of the institution will already be present. In that context, the image is "extra" and may not need to be identified to a screen-reader. It shouldn't be redundant with text that's already present. Maybe just identified as "decorative" in those contexts? I'm not sure.


**whether we omit display of logos at some viewport or window width**

Yes, definitely. The logo would be the first element to drop out as the window gets narrow. Don't know what that breakpoint is yet.


**whether we limit horizontal width in some way**

Yes. Some kind of max-width and max-height magic. 

In many cases the institution has a small version of the logo, but that's not the one they've provided in the manifest. :-(  
<img width="49" alt="biblissima" src="https://user-images.githubusercontent.com/6945691/52360248-6e3ea580-29f0-11e9-8a2d-7d6b0b35dd21.png">
<img width="94" alt="bnf" src="https://user-images.githubusercontent.com/6945691/52360256-71d22c80-29f0-11e9-809b-face712f2099.png">
<img width="151" alt="harvard" src="https://user-images.githubusercontent.com/6945691/52360410-caa1c500-29f0-11e9-9436-b5638cc3f958.png">

**Comment by sdellis** [comment]
@jvine I agree that a the IIIF logo property is ambiguous in meaning without a requiredStatement. But it's hard for me to tell if there is a conflict with [the specs](https://iiif.io/api/presentation/3.0/#logo) if we consider it "decorative" in the top bar, but display it in the initially hidden information side panel? Here's the wording:

> `logo` A small external image resource that represents an individual or organization associated with the resource that has the logo property. This could be the logo of the owning or hosting institution. The logo must be clearly rendered when the resource is displayed or used, without cropping, rotating or otherwise distorting the image.

**Comment by jvine** [comment]
No, it's not decorative in the top bar. It's decorative when it is accompanied by a clear text presentation of the institution's name, for example in a display like this:

<img width="240" alt="screen shot 2019-02-06 at 10 23 58 am" src="https://user-images.githubusercontent.com/6945691/52364130-591a4480-29f9-11e9-8a21-7a3918094cd7.png">

where you would not want the screen reader to render the organization name twice - once for the logo alt text and once for the actual text. IMO, the logo is not conveying additional information in this context, while it IS in the header bar. I want to be context specific with alt text, and keep it meaningful and not intrusive.
